src/main/java/gift/domain/user/UserController.java

```diff
@@ -1,15 +1,29 @@
 package gift.domain.user;
 
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
 
 @Controller
 @RequestMapping("/users")
 public class UserController {
 
+    private static final String KAKAO_AUTH_URL = "https://kauth.kakao.com/oauth/authorize";
+    private static final String RESPONSE_TYPE = "code"; // 기본적으로 "code"로 설정
+    @Value("${kakao.client.id}")
```

Comment:
해당 값들은 하나의 단어를 의미하여 일반적으로 다음과 같이 표현합니다!
```suggestion
    @Value("${kakao.client-id}")
    private String clientId; // 카카오 개발자 콘솔에서 발급받은 클라이언트 ID
    @Value("${kakao.redirect-url}")
    private String redirectUri; // 카카오 로그인 후 리다이렉트될 URI
```

Reply:
이 부분이 저도 미묘하게 신경쓰였습니다. . 으로 구분하시는 분도 계시고 - 로 구분하시는 분도 계셨는데
덕분에 올바른 표기법에 대해 알게 되었습니다. 감사합니다!